### PR TITLE
[stari_task0]: HelloWorld pr.

### PR DESCRIPTION
**Pull request by gyfffffff**
### What?

- brief: C++ environment and pr test. 
- author: charon0707@163.com

### What changes are included in this PR?

upload hello.cpp.
upload test screencut.

### Are these changes tested?
testcode

### CODE REVIEW

File: stl/stari/hello.exe
Comment (Light-City) [inline stl/stari/hello.exe:1]: this file should add into .gitignore or remove

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by gyfffffff** [comment]
> The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

I have fixed it, thank u.

File: stl/stari/gyf/src/hello.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main(void)
+{
```
Comment (martinx) [inline stl/stari/gyf/src/hello.cpp:4]: Please write code according to Google Code Style

1: `{` do not need to start a new line
2: Indent two spaces

```cpp
int main(void)
{
    std::cout << "Hello World" << std::endl;
    return 0;
}
```


=> 

```cpp
int main(void) {
  std::cout << "Hello World" << std::endl;
  return 0;
}
```

Comment (gyfffffff) [reply stl/stari/gyf/src/hello.cpp]: > Please write code according to Google Code Style
> 
> 1: `{` do not need to start a new line 2: Indent two spaces
> 
> ```c++
> int main(void)
> {
>     std::cout << "Hello World" << std::endl;
>     return 0;
> }
> ```
> 
> =>
> 
> ```c++
> int main(void) {
>   std::cout << "Hello World" << std::endl;
>   return 0;
> }
> ```

The style has been fixed, thank u!

**Comment by martinx** [comment]
Your team and members do not match the README.md 😄

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**